lib/json.ps1

```diff
@@ -146,19 +148,58 @@ function normalize_values([psobject] $json) {
     # Iterate Through Manifest Properties
     $json.PSObject.Properties | ForEach-Object {
 
+        $value = $_.Value
+        # Recursively edit psobjects
+        # If the values is psobjects, its not normalized
+        # For Example if manfiest have architecture and it's architecture have array with single value it's not formated
+        # @see: Without recursion: https://i.imgur.com/pHOf9f2.png
```

Comment:
Please change these to images with GitHub URL you provided in PR, just to have it in one place 😄

Reply:
I rather post there link to PR, instead of images. There were only reminders to conversation / review.
`# @see https://github.com/lukesampson/scoop/pull/2642#issue-220506263` would be better in my opinion.